Project/Assets/Editor/Annotations/AnnotationInteractionEditor.cs

```diff
@@ -143,4 +188,40 @@ void GetImagePath()
         }
         GUILayout.EndHorizontal();
     }
+
+
+    /// <summary>
+    /// Allows the user to save any edits made to the full annotation to a text file
+    /// </summary>
+    void SaveTextFileButton()
+    {
```

Comment:
> doesn't need to be resolved in this PR

I am a fan of this function.

This could be further improved by having the `Annotation` automatically link to the saved text file.